Assets/MixedRealityToolkit/MSBuild/MSBuildTemplates/SDKProjectTemplate.csproj

```diff
@@ -0,0 +1,119 @@
+<Project>
+  <PropertyGroup>
+    <!--anborod: This is a weird thing, it is a required property (even if commented)-->
+    <!--<TargetFrameworks>netstandard2.0;uap10.0;net46</TargetFrameworks> -->
+    
+    <ProjectGuid><!--PROJECT_GUID_TOKEN--></ProjectGuid>
+    <AllowUnsafeBlocks><!--ALLOW_UNSAFE_TOKEN--></AllowUnsafeBlocks>
+    <LangVersion><!--LANGUAGE_VERSION_TOKEN--></LangVersion>
+    
+    <DevelopmentBuild><!--DEVELOPMENT_BUILD_TOKEN--></DevelopmentBuild>
+    
+    <RelativeBuildFolder><!--OUTPUT_PATH_TOKEN--></RelativeBuildFolder>
+    <IsEditorOnlyTarget><!--IS_EDITOR_ONLY_TARGET_TOKEN--></IsEditorOnlyTarget>
+    <UnityEditorInstallFolder><!--UNITY_EDITOR_INSTALL_FOLDER--></UnityEditorInstallFolder>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Platform)' == ''">
+    <Platform><!--DEFAULT_PLATFORM_TOKEN--></Platform>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Configuration)' == ''">
+    <Configuration>InEditor</Configuration>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <Configurations>InEditor;Player</Configurations>
+    <Platforms><!--SUPPORTED_PLATFORMS_TOKEN--></Platforms>
+    <UnityPlatform>$(Platform)</UnityPlatform>
+    <UnityConfiguration>$(Configuration)</UnityConfiguration>
+  </PropertyGroup>
+
+  <!-- This is special cased here for WSA player only-->
```

Comment:
><!-- This is special cased here for WSA player only--> [](start = 2, length = 54)

Is this comment supposed to be for this line for the one below that looks specifically for WSA player?

Reply:
Will update.

---
In reply to: [301778494](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5215#discussion_r301778494) [](ancestors = 301778494)